dag.yml

```diff
@@ -118,6 +118,8 @@ steps:
     - data://garden/cait/2022-08-10/ghg_emissions_by_sector
 
   # Autogenerated steps by walkthrough
+
```

Comment:
I was thinking that, just to keep the dag cleaner, and to avoid conflicts, maybe the walkthrough steps could be in their own dag in dag_files.

Reply:
Great idea, fixed.